packages/components/icons/src/generated/AngleDownReact.tsx

```diff
@@ -133,29 +113,24 @@ export const getIconStyles = (props: Props) => css`
   }
   flex-shrink: 0;
 `;
-
 const SvgAngleDown = (props: SVGProps) => (
   <svg
     width={24}
     height={24}
     viewBox="0 0 24 24"
     xmlns="http://www.w3.org/2000/svg"
-    role="img"
```

Comment:
I think icons should not have changes in this PR.
Perhaps you have an outdated version of `@svgr/cli` installed?
It should be version `6.5.1`.

Reply:
I'm using the `@svgr/cli` version 6.5.1. 
I'm not sure why there are changes still

Reply:
Here is the commit -> [5602bb9](https://github.com/commercetools/ui-kit/pull/2375/commits/5602bb924cbebb9ea9f23beee9db384637fb4f9d)